Xamarin.Forms.Controls/GalleryPages/CollectionViewGalleries/CarouselViewGalleries/ExampleTemplateCarousel.xaml.cs

```diff
@@ -56,7 +58,15 @@ public ExampleTemplateCarousel()
 					}
 				}
 			};
-			GestureRecognizers.Add(gesture);
+			//GestureRecognizers.Add(gesture);
```

Comment:
Adding a gesture to the Item template breaks scrolling.

Reply:
Do we have an open issue for that yet?